lib/tasks/import_prebakedstats.rake

```diff
@@ -0,0 +1,160 @@
+require 'csv'
+
+namespace :import do
+
+  desc "import CSV data into database"
+  task :prebakedstats, [:csv_file] => [:environment] do
+    habitats = Habitat.all
+    geo_entity_types = ["countries", "regions"].freeze
+
+    # import habitat data CSVs for each entity type
+    geo_entity_types.each do |geo_entity_type|
+      habitats.each do |habitat|
+        puts habitat.name
+        csv_file = "#{geo_entity_type}/#{habitat.name}.csv"
+
+        puts csv_file
+        import_new_csv_file(geo_entity_type, habitat.name, csv_file)
+      end
+    end
+  end
+
+  def import_new_csv_file(geo_entity_type, habitat, csv_file)
+    filename = "#{Rails.root}/lib/data/#{csv_file}"
+    csv = File.open(filename, encoding: "utf-8")
+    csv_headers = File.readlines(csv).first.split(",")
+
+    CSV.parse(csv, headers: true, encoding: "utf-8") do |row|
+      csv_new_row = row.to_hash
+
+      if csv_headers.grep(/baseline/).any?
+        puts csv_new_row.inspect
+        parse_change_stat csv_headers, csv_new_row, geo_entity_type, habitat
+      else
+        puts csv_new_row.inspect
+        parse_geo_entity_stat csv_headers, csv_new_row, geo_entity_type, habitat
+      end
+    end
+  end
+
+  def parse_change_stat csv_headers, csv_row, geo_entity_type, habitat
+    total_value_years = {}
+ 
+    parse_change_stat_hash = {
+      iso3: csv_headers[0],
```

Comment:
if the headers are already following the column names definition, you could use `slice` or the entire hash.
You can check the `import_redlist_data.rake` for reference in general.